tests/integration/help_test.py

```diff
@@ -0,0 +1,109 @@
+"""Various tests for `candies.cli.command.Command.help`."""
+
+import sys
+from typing import Callable, Tuple
+
+from candies.cli.args.arg import Arg
+from candies.cli.cli import cli
+from candies.cli.command import Command
+
+
+class Output:
+    """Overrides the default `stdout`."""
+
+    def __init__(self):
+        self.text = ''
+
+    def write(self, text: str):
+        self.text += text
+
+
+def execute(cli_: Callable, with_: str) -> Tuple[Output, SystemExit]:
+    stdout = sys.stdout
+    output = sys.stdout = Output()
+    error = None
+
+    try:
+        cli_(with_.split())
+    except SystemExit as ex:
+        error = ex
+
+    sys.stdout = stdout
+
+    return output, error
+
+
+def test_default_help_without_description():
+    # Arrange.
+    @cli
+    def main():
+        pass
+
+    # Act.
+    output, error = execute(main, with_='--help')
+
+    # Assert.
+    assert error.code == 0
```

Comment:
Is it possible to have `error.code != 0` here, if `output.text.startswith('usage: main')` is `True`?

Reply:
Totally removed these checks.